server/src/repositories/person.repository.ts

```diff
@@ -49,8 +51,22 @@ export class PersonRepository implements IPersonRepository {
     await this.personRepository.clear();
   }
 
-  async deleteAllFaces(): Promise<void> {
-    await this.assetFaceRepository.query('TRUNCATE TABLE asset_faces CASCADE');
+  async deleteAllFaces({ sourceType }: DeleteAllFacesOptions): Promise<void> {
+    if (sourceType === undefined) {
+      await this.assetFaceRepository.query('TRUNCATE TABLE asset_faces CASCADE');
```

Comment:
might be worth using a switch statement here

Reply:
@jrasm91, now using ENUM for sourceType it's simpler and don't think require switch unless you tell me otherwise

Reply:
Sounds good.